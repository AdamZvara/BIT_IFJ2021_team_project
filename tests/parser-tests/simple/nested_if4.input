require "ifj21"

global bar : function(integer) : integer

function bar(i : integer) : integer
	
	write("bar","\n")
	return i
end

function main()
	local i : integer = 0
	while i < 15 do 
		if i +  5 > 15 then
			local a : integer = i
			write(a," ")
		else
			if i < 10 then
				i = bar(i)
			else
				i = i + 1
			end
		end

		i = i + 1
	end
end

main()
