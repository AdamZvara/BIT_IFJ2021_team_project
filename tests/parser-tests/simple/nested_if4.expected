bar
bar
bar
bar
bar
bar
bar
bar
bar
bar
12 13 14 